### ISSUE TITLE
Unable to select drop down using Voice Control

### ISSUE THREAD

**Issue by kylebennett-sage** [Sage/carbon#4470]
### Current behaviour

Dropdown elements cannot be selected with VoiceOver commands.

### Expected behaviour

Dropdown Elements should be selectable with VoiceOver commands.

### CodeSandbox or Storybook URL

https://codesandbox.io/s/dropdown-voiceover-xi9ex

### JIRA Ticket (Sage Only)

RIV-532

### Carbon Version

87.0.1

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What Operating System are you seeing the problem on?

MacOS, Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by msinga** [comment]
RIV-532

**Comment by v99rly** [comment]
I've had a look at this when using VoiceOver and it seems to work ok. When using the modifier key you tab into the drop down and then use the arrow keys to highlight the option you want, you can then use the enter key to select that option.

**Comment by samtjo** [comment]
Will be fixed as part of FE-4300 - @grabkowski

**Comment by sianford** [comment]
FE-4300 did not fix this issue. Logged FE-4925 for this.

**Comment by robinzigmond** [comment]
After some investigation, we do not believe there are any major issues with the usability of the Carbon Select with Voice Control. Please note the following (to compare behaviour with vanilla selects see [https://playground.tempertemper.net/forms/select/)](https://playground.tempertemper.net/forms/select/):

- Find Next and Select Next should not do anything, and do not with a vanilla select. These commands are to do with finding/selecting text on the whole page, not anything to do with selecting from a list. ("Find Next" selecting Green in the Carbon storybook example - and more generally selecting the first item beginning with a G - is a bug in the Carbon component, which should be fixed, but this does not impact usability in a significant way)
- to select an item with Voice Control, do the following:
  - first focus the field. This can be done by one or any of:
    - "click {label name}"
    - using "press tab key" or "next field"/"previous field" to navigate around
    - "show numbers" followed by reading out the number associated to the field
  -  if the dropdown options are not yet open, you can do so by saying "press space key" (this doesn't always work, but the same problem seems to exist on vanilla HTML selects!) or "move down" (which seems to always work)
  - to select and navigate within the list of options, you can:
    - use the commands "move down" and "move up" to move through the list one item at a time (this also wraps from one end to another)
    - say "click {item name}" to select a visible option directly (this seems to not always work - eg it does on CodeSandbox but not Storybook)
    - say "show numbers" again, this should label each visible item with a number than can then be called out
    
There are a few small issues still to fix - notably, that Find Next sometimes selects an item, and that "scroll up/down" commands do not scroll through the list of items when a larger scrollable list is open - and we will raise a new ticket for these. But please let us know @kylebennett-sage if with the above instructions you consider the original issue resolved?
